sunpy/io/special/asdf/tags/map/generic_map.py

```diff
@@ -19,7 +18,7 @@ class GenericMapType(SunPyType):
     def from_tree(cls, node, ctx):
         # Use the factory here to get the correct subclass back
         out_map = sunpy.map.Map(np.asarray(node['data']), node['meta'])
-        out_map.shift(*node['shift'])
+        out_map.shift_reference_coord(*node['shift'])
```

Comment:
You need to guard this as new files wont have shift in the node.
```suggestion
        if 'shift' in node:
            out_map.shift_reference_coord(*node['shift'])
```

Reply:
🤦 thanks!